### ISSUE TITLE
Adding observation to projects failing silently

### ISSUE THREAD

**Issue by loarie** [inaturalist/inaturalist#617]
Clicking 'add' on this observation 
http://www.inaturalist.org/observations/1706252
to this project
http://www.inaturalist.org/projects/apff-cuatro-cienegas-coahuila
is failing silently
![screen shot 2015-06-29 at 2 29 12 pm](https://cloud.githubusercontent.com/assets/598026/8418942/2c02e9d6-1e6b-11e5-84a6-f7e93551afac.png)
This might be related
https://groups.google.com/forum/#!topic/inaturalist/lryEMEka0WE

**Comment by kueda** [comment]
Not quite done: still fails silently if there is a place rule, maybe a taxon rule.

**Comment by kueda** [comment]
I wasn't actually able to replicate that remaining bug. I thought it was happening when I wrote that, but now it doesn't seem to be. The problem with the specific project and obs you listed here is that the project had a place rule for a place that had been deleted. I added a check a while back to prevent this, but I think I only prevented place deletion for place that were just associated with projects, not ones that were associated through project observation rules. Anyway, I changed that in b151f11
